composer.lock

```diff
@@ -4,24 +4,24 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "f382af9a56f6fd0ebf0ff4745ed03d8f",
+    "content-hash": "7116684c64d03de90f34468c26398fe0",
     "packages": [
         {
             "name": "apimatic/core-interfaces",
-            "version": "0.1.0",
+            "version": "dev-3-php-8.2-support",
```

Comment:
run composer update locally after releasing core-interfaces, to update composer lock file